sunpy/physics/tests/test_differential_rotation.py

```diff
@@ -112,3 +136,31 @@ def test_solar_rotate_coordinate():
     np.testing.assert_almost_equal(d.observer.lat.to(u.deg).value, 7.248, decimal=3)
     np.testing.assert_almost_equal(d.observer.radius.to(u.AU).value, 1.006954, decimal=6)
     assert isinstance(d.observer, frames.HeliographicStonyhurst)
+
+
+def test_warp_sun():
+    pass
```

Comment:
it's a placeholder, hopefully someone will write it! (also I missed this comment and cba to fix it now)